FirebaseAuth/Sources/Swift/AuthProvider/PhoneAuthProvider.swift

```diff
@@ -152,17 +155,10 @@ import Foundation
     open func verifyPhoneNumber(with multiFactorInfo: PhoneMultiFactorInfo,
```

Comment:
The async and non-async versions of `verifyPhoneNumber` should share an implementation.  See how this is done for the other public APIs.

Reply:
Done